curriculum/challenges/english/25-front-end-development/workshop-cafe-menu/5f3477ae9675db8bb7655b30.md

```diff
@@ -21,7 +21,7 @@ assert(hasH1);
 You should not add a new `style` tag. Add the new CSS rules to the existing `style` tag.
 
 ```js
-const hasManyStyleTags = document.querySelectorAll('style').length > 1;
+const hasManyStyleTags = document.querySelectorAll('style').length > 2;
 assert(!hasManyStyleTags);
```

Comment:
```suggestion
assert.isAtMost(document.querySelectorAll('style').length, 2);
```

Reply:
Suggested changes have been taken in to account, thank you for your cleaner, and more simplified approach.